Index: includes/theme.inc
===================================================================
--- includes/theme.inc	(revision 204)
+++ includes/theme.inc	(working copy)
@@ -1728,6 +1728,16 @@
   }
 }
 
+function drupal_get_rdf_namespaces() {
+  // Serialize the RDF namespaces used in RDFa annotation.
+  $xml_rdf_namespaces = array();
+  foreach (module_invoke_all('rdf_namespaces') as $prefix => $uri) {
+    $xml_rdf_namespaces[] = 'xmlns:' . $prefix . '="' . $uri . '"';
+  }
+  return implode("\n  ", $xml_rdf_namespaces);
+}
+
+
 /**
  * Process variables for page.tpl.php
  *
@@ -1816,6 +1826,8 @@
   $variables['scripts']           = drupal_get_js();
   $variables['tabs']              = theme('menu_local_tasks');
   $variables['title']             = drupal_get_title();
+  $variables['rdf_namespaces']    = drupal_get_rdf_namespaces();
+  $variables['grddl_profile']     = 'http://ns.inria.fr/grddl/rdfa/';
   // Closure should be filled last.
   $variables['closure']           = theme('closure');
 
Index: themes/garland/page.tpl.php
===================================================================
--- themes/garland/page.tpl.php	(revision 203)
+++ themes/garland/page.tpl.php	(working copy)
@@ -1,9 +1,9 @@
 <?php
 // $Id: page.tpl.php,v 1.18 2008/01/24 09:42:53 goba Exp $
-?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
-  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php print $language->language ?>" lang="<?php print $language->language ?>" dir="<?php print $language->dir ?>">
-  <head>
+?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php print $language->language ?>" 
+  <?php print $rdf_namespaces ?>>
+<head profile="<?php print $grddl_profile ?>">
     <title><?php print $head_title ?></title>
     <?php print $head ?>
     <?php print $styles ?>
