### Eclipse Workspace Patch 1.0
#P neologism2
Index: sites/all/modules/rdf/rdf.api.inc
===================================================================
--- sites/all/modules/rdf/rdf.api.inc	(revision 410)
+++ sites/all/modules/rdf/rdf.api.inc	(working copy)
@@ -295,8 +295,27 @@
   return preg_match('/^\[?[\w\-]+:[\w\-]*\]?$/', (string)$curie); // FIXME
 }
 
+/*
+NCName           ::= NCNameStartChar NCNameChar*
+NCNameChar       ::= NameChar - ':'
+NCNameStartChar  ::= Letter | '_'
+
+QName ::= PrefixedName | UnprefixedName
+PrefixedName    ::= Prefix ':' LocalPart
+UnprefixedName  ::= LocalPart
+Prefix          ::= NCName
+LocalPart       ::= NCName
+
+NCName ::= NCNameStartChar = ^[a-zA-Z_]+ NCNameChar* = [a-zA-Z0-9_]* == ^[a-zA-Z_]+[a-zA-Z0-9_]*
+PrefixedName = ^[a-zA-Z_]+[a-zA-Z0-9_]*:^[a-zA-Z_]+[a-zA-Z0-9_]*
+QName =  (^[a-zA-Z_]+[a-zA-Z0-9_]*:^[a-zA-Z_]+[a-zA-Z0-9_]*) | (^[a-zA-Z_]+[a-zA-Z0-9_]*)
+
+*/
 function rdf_is_valid_qname($qname) {
-  return preg_match('/^[\w\-]+:[\w\-]+$/', (string)$qname); // FIXME
+  //return preg_match('/^[\w\-]+:[\w\-]+$/', (string)$qname); // FIXME
+  $result = preg_match('/^[a-zA-Z_]+[a-zA-Z0-9_-]*:[a-zA-Z0-9_]*/', $qname) || preg_match('/^[a-zA-Z_]+[a-zA-Z0-9_]*/', $qname);
+  return $result;
+  //return preg_match('/(^[a-zA-Z_]+[a-zA-Z0-9_]*:^[a-zA-Z_]+[a-zA-Z0-9_]*) | (^[a-zA-Z_]+[a-zA-Z0-9_]*)/', (string)$qname); 

 }