Index: rdfcck/rdfcck.module
===================================================================
--- rdfcck/rdfcck.module	(revision 358)
+++ rdfcck/rdfcck.module	(working copy)
@@ -299,8 +299,8 @@
     case NULL:
     case RDF_SITE_URI:
       // TODO improve the discovery mechanism for passing the nid to the loading function.
-      //$uri = url('node/' . arg(1), array('fragment' => 'this', 'absolute' => TRUE));
-      $uri = url('node/' . arg(1), array('absolute' => TRUE));
+      $uri = url('node/' . arg(1), array('fragment' => 'this', 'absolute' => TRUE));
+      //$uri = url('node/' . arg(1), array('absolute' => TRUE));
       return array($uri => new RDF_QueryCallback('rdfcck_load_node', array(arg(1))));
   }
 }
Index: rdf/rdf.pages.inc
===================================================================
--- rdf/rdf.pages.inc	(revision 358)
+++ rdf/rdf.pages.inc	(working copy)
@@ -18,7 +18,7 @@
 }
 
 function rdf_export_entity($type, $id) {
-  $data = rdf_query(url($type . '/' . $id, array('absolute' => TRUE)));
+  $data = rdf_query(url($type . '/' . $id, array('fragment' => 'this', 'absolute' => TRUE)));
   rdf_export($data, implode('-', array($type, $id)));
 }
 
